plugins/CoreAdminHome/templates/whatIsNew.twig

```diff
@@ -0,0 +1,24 @@
+<div piwik-content-intro>
+    <h2>{{ 'CoreAdminHome_WhatIsNewTitle'|translate|e('html_attr') }}</h2>
+</div>
+<div class="whatisnew">
+
+    {% for change in changes %}
+
+        <div class="card">
+            <div class="card-content">
+                <span style="float: left; font-size:32px; color:#3450A3; margin-right:10px" class="icon-new_releases"></span>
+                <h2 class="card-title">{{ change.plugin_name }} - {{ change.title }}</h2>
+                {{ change.description | raw }}
+                {% if not change.link is empty %}
+                <p>
+                    <br>
+                    <a href="{{ change.link }}" target="_blank">{{ change.link_name }}</a>
```

Comment:
It's important to escape html attributes to ensure it doesn't break the html. Also for potential external links we should still use `rel="noopener"`
```suggestion
                    <a href="{{ change.link|e('html_attr') }}" target="_blank" rel="noopener">{{ change.link_name }}</a>
```

Reply:
Noted and fixed.